### PR TITLE
Added API docs

### PR DESCRIPTION
**Pull request by thecodeassassin**
Added API docs to documentation

### CODE REVIEW

**Comment by thecodeassassin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #12** <a href="https://app.graphite.dev/github/pr/symbiosis-cloud/symbiosis-docs/12" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/symbiosis-cloud/symbiosis-docs/12?utm_source=stack-comment).

**Review by thecodeassassin — COMMENTED** [review]
@Styren Should we hide certain routes (like strip-webhook etc) from the OpenAPI spec?

**Review by Styren — APPROVED** [review]
Wow looks great!!! But lets clean up the api docs/add more content before we merge this.

File: docusaurus.config.js [hunk 1]
```diff
@@ -48,7 +67,19 @@ const config = {
         src: 'img/logo.png',
         target: '_self',
       },
-      items: [],
+	  items: [
+		{
+		  type: 'doc',
+		  docId: 'overview/intro',
+		  position: 'left',
+		  label: 'Documentation',
```
Comment (Styren) [inline docusaurus.config.js]: Maybe just docs?
```suggestion
		  label: 'Docs',
```

**Pull request merged**